src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DocumentSynchronization/RazorDidSaveTextDocumentEndpoint.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts;
+using Microsoft.CodeAnalysis.Razor;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.DocumentSynchronization;
+
+internal class RazorDidSaveTextDocumentEndpoint : IVSDidSaveTextDocumentEndpoint
+{
+    public bool MutatesSolutionState => false;
```

Comment:
```suggestion
    public bool MutatesSolutionState { get; } = false;
```

Reply:
I'm not fussed, but I'm curious as to why. You don't like expression bodied properties? You like fields?

Reply:
This particular case is totally legit. However, a getter-only property and an expression-bodied property are definitely not equivalent.

The reason that I always look for these is because the IDE's "Implement Interface" and "Implement Abstract Class" code fixes always produce expression-bodied properties so they can include a `throw new NotImplementedException()`. So, if somebody implementing an interface quickly, it's easy to accidentally put something expensive on the right side of the `=>`:

```C#
public class C : I
{
    public MyBigObject Obj => new MyBigObject();
}
```

I've run into this sort of thing a couple of times in performance investigations, so I always watch out for unintended expression-bodied properties. However, like I said, this case is totally fine. The JIT will inline it and all will be good.

Reply:
Yeah, I've definitely also seen people fall into that trap, and so its something to watch for. I guess I wouldn't bother to change it unless it was a problematic example, but as I said, I'm not fussed. In general the `.editorconfig` in razor is not super opinionated, and we should probably slowly work to make it so, just to avoid the PR noise.